relay-server/src/service.rs

```diff
@@ -116,6 +118,15 @@ impl ServiceState {
         let system = System::current();
         let registry = system.registry();
 
+        let runtime = tokio::runtime::Builder::new_multi_thread()
+            .enable_all()
+            .on_thread_start(clone!(system, || System::set_current(system.clone())))
+            .build()
```

Comment:
Please use a single thread here, either through the single-threaded runtime, or through configuring the number of workers. The healthcheck shouldn't spawn that many threads.

For actual multi-threaded runtimes later, we should use [`Config::cpu_concurrency`](https://getsentry.github.io/relay/relay_config/struct.Config.html#method.cpu_concurrency).